dashboard/app/templates.html

```diff
@@ -98,6 +98,7 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 		<th><a onclick="return sortTable(this, 'Count', numSort)" href="#">Count</a></th>
 		<th><a onclick="return sortTable(this, 'Last', timeSort)" href="#">Last</a></th>
 		<th><a onclick="return sortTable(this, 'Reported', timeSort)" href="#">Reported</a></th>
+                <th><a onclick="return sortTable(this, 'Activity', textSort)" href="#">Activity</a></th>
```

Comment:
Please use tabs instead of spaces as all other code around does.

Reply:
My bad, will do.